infrastructure/zk/src/validate_migrations.ts

```diff
@@ -0,0 +1,39 @@
+import * as utils from './utils';
+import fs from 'fs';
+
+async function validateMigration(filePath: string) {
+    const content = await fs.promises.readFile(filePath, { encoding: 'utf-8' });
+    const isBackwardsCompatible = !content.includes('RENAME') && !content.includes('DROP');
```

Comment:
are these checks case sensitive? sql is case insensitive.

Reply:
corrected